### PR TITLE
[Fix][httpKernel] Wrong links

### PR DESCRIPTION
**Pull request by 20uf**
Hello here,

### Problem
Fixes bad links pointing to the HttpKernel api.
http://symfony.com/doc/current/components/http_kernel.html#the-kernel-request-event

Happy New Year !

### CODE REVIEW

File: components/http_kernel.rst [hunk 1]
```diff
@@ -124,9 +124,9 @@ For general information on adding listeners to the events below, see
 
 .. caution::
 
-    As of 3.1 the :class:`Symfony\\Component\\Httpkernel\\HttpKernel` accepts a
+    As of 3.1 the :class:`Symfony\\Component\\HttpKernel` accepts a
```
Comment (javiereguiluz) [inline components/http_kernel.rst]: I'd say that the original reference was correct. See: http://api.symfony.com/3.1/Symfony/Component/HttpKernel/HttpKernel.html

File: components/http_kernel.rst [hunk 2]
```diff
@@ -124,9 +124,9 @@ For general information on adding listeners to the events below, see
...
     fourth argument, which must be an instance of
-    :class:`Symfony\\Component\\Httpkernel\\Controller\\ArgumentResolverInterface`.
+    :class:`Symfony\\Component\\HttpKernel\\Controller\\ArgumentResolverInterface`.
```
Comment (javiereguiluz) [inline components/http_kernel.rst]: Nice catch!

**Review by javiereguiluz — CHANGES_REQUESTED** [review]

**Comment by 20uf** [comment]
Indeed I have done wrong.

**Review by javiereguiluz — APPROVED** [review]
👍 

Very nice catch! Thanks @20uf.

**Comment by xabbuh** [comment]
Thank you @20uf.

**Pull request closed without merging**